doppelganger/bayesnets.py

```diff
@@ -96,7 +97,7 @@ def to_json(self):
         blob['type_to_network'] = {
             type_: json.loads(network.to_json()) for type_, network in self.type_to_network.items()
         }
-        return json.dumps(blob, indent=4, sort_keys=True)
+        return str(json.dumps(blob, indent=4, sort_keys=True, ensure_ascii=False))
```

Comment:
This is my bad.... I was getting some encoding errors that I had put this in for but now I can't reproduce them. Apologies! Do I need to resubmit the pull request to take this out?